### PR TITLE
Remove old content in preparation for new version

### PR DESCRIPTION
**Pull request by cattsmall**
## Description
In this PR, All irrelevant files have been deleted. If we need any of the old files, they are accessible via the gh-pages branch. This branch will be empty (minus documentation files and favicons) so the new files can be added.

## Context

Related to roadmap item(s):

- #3 

Additional changes included in this pull request:
- Renamed `files` folder as `storage` and added a `pdf` folder. Do we need a CDN? If we get one, I'll remove that file!

## Screenshots
image
image

## Reviewers

@GoodForPoC/owners @GoodForPoC/moderators

### CODE REVIEW

**Review by 0xStarcat — APPROVED** [review]
good to merge~

File: CNAME [hunk 1]
```diff
@@ -1 +0,0 @@
-goodforpocin.tech
```
Comment (cattsmall) [inline CNAME]: Don't need this because it was for GitHub Pages

File: Gemfile [hunk 2]
```diff
@@ -1,8 +0,0 @@
-source 'https://rubygems.org'
-
-group :jekyll_plugins do
-  gem 'github-pages'
-  gem 'jekyll-paginate'
-  gem 'jekyll-assets'
-  gem 'jekyll-contentblocks'
-end
```
Comment (cattsmall) [inline Gemfile]: Not sure if this file is needed since we're killing Jekyll.

File: Gemfile.lock [hunk 3]
```diff
@@ -1,149 +0,0 @@
-GEM
```
Comment (cattsmall) [inline Gemfile.lock]: Not sure we need this massive file. It was also just for Jekyll.

File: Gemfile (hunk 2 above)
Comment (0xStarcat) [reply Gemfile]: the replacement should auto-generate a new Gemfile with the Gems needed for that project. you're correct that this is only needed for Jekyll

**Review by jalcine — APPROVED** [review]

**Pull request merged**